### PR TITLE
Almost completed SignUpActivity.java

### PR DESCRIPTION
**Pull request by manojsuthar123**
If you have any doubt regarding code then please contact me :-)

### CODE REVIEW

File: SignUpActivity.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    DatabaseReference mDatabase;
+    FirebaseUser mCurrentUser;
+    String mEmail;
+    String mName;
+    String mDepartmant;
+    String mYear;
+    String mRegNumber;
+    String mCollege;
+    String mPassword;
+    String mConfirmPassword;
```
Comment (abhi935235) [inline SignUpActivity.java:26]: no issues for now but conventionally it should be uEmail,uName  and so on and instances should be start with mName

File: SignUpActivity.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    private EditText EmailEditText;
+    private EditText NameEditText;
+    private EditText DepartmantEditText;
+    private EditText YearEditText;
+    private EditText CollegeEditText;
+    private EditText RegNumberEditText;
+    private EditText ClassEditText;
+    private EditText PasswordEditText;
+    private EditText ConfirmPasswordEditText;
+    private Button SubmitButton;
```
Comment (abhi935235) [inline SignUpActivity.java:37]: it should be named as mEmailEditText and so on , anyways no issues for now 
Great work

Comment (manojsuthar123) [reply SignUpActivity.java:37]: Ok I will resolve this problem in next pull

**Review by abhi935235 — COMMENTED** [review]
No problem just take care when you work on your next activity.

**Pull request merged**